src/Microsoft.TestPlatform.Utilities/Microsoft.TestPlatform.Utilities.csproj

```diff
@@ -20,6 +20,11 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Xml" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
```

Comment:
Not ideal :-/ This will require all the nuget packages to declare an extra dependency.

Reply:
Please remove it.